### ISSUE TITLE
NVDA not announcing aria-selected=true attribute for the item

### ISSUE THREAD

**Issue by yashank676** [nvaccess/nvda#17373]
### Steps to reproduce:
[code pen](https://codepen.io/Yashank-Aggarwal/pen/LYwXGZW)

1. open codepen
2. click on or go to the rendered area in lower half of screen
3. use tab to go inside the list items

### Actual behavior:
NVDA is not announcing the `aria-selected=true` attribute for `Dashboard list-item`

### Expected behavior:
NVDA should announce the `selected` attribute for the list-item

#### NVDA version:
`2024.4 (2024.4.0.34423)`
#### Windows version:
Windows 11 (Version: 23H2) (OS build: 22631.4317)
#### Name and version of other software in use when reproducing the issue:
Google Chrome, Version 129.0.6668.59 (Official Build)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
In general NVDA announces when something is not selected as „not selected“ e.g. in a multi select list. But when something is selected, NVDA is only announcing the item without the word „selected“ which is consistent with all other places in Windows such as e.g. file explorer. So reporting „selected“ is redundant verbosity. Closing as wont fix.

**Comment by yashank676** [comment]
in a multi select list if one of the item is selected and others are not selected then while navigation NVDA it doesn't announce any state whether selected or not.

**Comment by Adriani90** [comment]
Ok this seems to impact the tree role only, this is a duplicate of #14352 and therefore it can stay closed.
